openstack/blockstorage/v1/apiversions/urls.go

```diff
@@ -11,5 +12,7 @@ func getURL(c *gophercloud.ServiceClient, version string) string {
 }
 
 func listURL(c *gophercloud.ServiceClient) string {
-	return c.ServiceURL("")
+	u, _ := url.Parse(c.ServiceURL(""))
+	u.Path = "/"
+	return u.String()
```

Comment:
Try just returning `c.Endpoint` such as [here](https://github.com/rackspace/gophercloud/blob/master/openstack/orchestration/v1/apiversions/urls.go#L6) and [here](https://github.com/rackspace/gophercloud/blob/master/openstack/networking/v2/apiversions/urls.go#L10) and see if that works.